src/families/bitcoin/libcore-signOperation.js

```diff
@@ -59,6 +59,8 @@ async function signTransaction({
     additionals.push("decred");
   } else if (currency.id === "stealthcoin") {
     additionals.push("stealthcoin");
+  } else if (currency.id === "zencash") {
+    additionals.push("zencash");
   }
```

Comment:
maybe for later, but i think we could replace some of these ifs by `additionals.push(currency.id)`